megago/megago.py

```diff
@@ -35,15 +33,21 @@
 except pkg_resources.DistributionNotFound:
     PROGRAM_VERSION = "undefined_version"
 
-GODAG_FILE_PATH = os.path.join(DATA_DIR, "go-basic.obo")
 
+def is_go_term(candidate):
+    """ Returns true if a given string could be a valid GO-term identifier. This function only checks if the string
+    satisfies the formatting of a GO-term identifier, not if it actually exists in the ontology!
 
-def is_go_term(string):
+    Parameters
+    ----------
+    candidate: A string for which we should verify if it's a valid GO-identifier.
+
+    Returns
+    -------
+    True if the string is indeed a valid identifier, False otherwise.
```

Comment:
```suggestion
    Parameters
    ----------
    candidate : str
        A string for which we should verify if it's a valid GO-identifier.

    Returns
    -------
    bool
```
Numpy-style 😎
Do you maybe know an easy way for validating code style conformitty? I have heard of such things but never used them myself. Would be a cool feature to add to our CI.

Reply:
We could use the Python linter for that. It's a tool that checks if the code adheres to specific rules that we set up. I'll take a look to implement a new Github action with these checks (we could maybe use this: https://github.com/marketplace/actions/python-lint)